action.yml

```diff
@@ -1,15 +1,65 @@
-name: Model Inference Test
-description: 'This is a test to confirm we can publish an action to github marketplace'
+name: Time To Merge Tool - Model Inference Test
+description: 'This is the github action to predict time to merge for a new pull request'
+author: 'redhat-et'
+
 inputs:
```

Comment:
I'm not 100% sure how we handle the env variables, but I don't think we need them in this file and the inference action yaml.

Reply:
Looking at a few existing action.yml files I see most of them are inputting these env variables in this file itself. https://github.com/actions/stale/blob/main/action.yml
However, if you suggest we can keep them in the inference.yaml only and move them to action later?

Reply:
no no. let's leave them in both places for now. But I think eventually we will be able to move them out of the inference.yaml

Reply:
Yeah, I agree with this. :+1: